<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserManagerWidget</class>
 <widget class="QWidget" name="UserManagerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Manager</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>User Manager</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>98</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="newUserButton">
       <property name="text">
        <string>New User</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeUserButton">
       <property name="text">
        <string>Remove User</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="changeUserPassword">
       <property name="text">
        <string>Change User Password</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>501</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>User List</string>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>481</width>
      <height>151</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>150</number>
    </attribute>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Username</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Employee Number</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Rights</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>mDavies</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>0023</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>User</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>jRopotar</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>0486</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>Admin</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>jKettle</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>0731</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>User</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QPushButton" name="exitUserManagerButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>350</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
